sample/base-library/build.gradle

```diff
@@ -0,0 +1,27 @@
+buildscript {
```

Comment:
I have decomposed the library of the sample into a base library and a library itself so that the sample also covers the case of inheritance across module. Feel free to revert if you don't want to.